src/AccordionPanel.jsx

```diff
@@ -0,0 +1,198 @@
+import React from 'react';
+import cx from 'classnames';
+
+import Chunk from './Chunk';
+import Flex from './Flex';
+import FlexItem from './FlexItem';
+import Icon from './Icon';
+
+export const PANEL_CLASS = 'accordionPanel';
+export const ACTIVEPANEL_CLASS = 'accordionPanel--active';
+
+
+/**
+ * @module AccordionPanel
+ */
+class AccordionPanel extends React.Component {
+	constructor(props){
+		super(props);
+
+		this.state = {
+			open: this.props.isOpen
+		};
+
+		this._handleToggle = this._handleToggle.bind(this);
+	}
+
+	/**
+	 * @returns {undefined}
+	 *
+	 * Updates state to toggle `AccordionPanel` open and closed
+	 */
+	_handleToggle(){
+		const newState = !this.state.open;
+		const { setClickedPanel } = this.props;
+
+		if (setClickedPanel) {
+			setClickedPanel(this);
+		}
+
+		this.setState({
+			height: `${newState * this.content.getBoundingClientRect().height}px`,
```

Comment:
Since this height calculation is made 3 times, I'd recommend making it a function that takes `isOpen` as an argument

```js
height: getHeight(isOpen),
```